porch/repository/pkg/git/testing.go

```diff
@@ -38,10 +38,6 @@ import (
 	"k8s.io/klog/v2"
 )
 
-const (
-	Main plumbing.ReferenceName = "refs/heads/main"
```

Comment:
Nice!  I was going to ask if it was possible to do this, but I've deleted that comment now :-)

Reply:
Thanks!